app/models/ring.rb

```diff
@@ -0,0 +1,19 @@
+class Ring
+  def initialize(redirection_to_link)
+    @oldest_redirection = Redirection.order(created_at: :desc).last
```

Comment:
Random question: Why do this as an instance variable assignment in the constructor vs. a private method?